src/main/java/ooo/oshi/foreign/windows/Kernel32Library.java

```diff
@@ -60,5 +67,30 @@ public static int getCurrentProcessId() {
 
     private static final MethodHandle getCurrentProcessId = methodHandle("GetCurrentProcessId",
             FunctionDescriptor.of(JAVA_INT));
+    
+    /**
+     * Retrieves the NetBIOS name of the local computer. This name is established at system startup, when the system
+     * reads it from the registry.
+     *
+     * @return the NetBIOS name of the local computer.
+     */
+    public static String getComputerName() {
+    	try {
+        	SegmentAllocator allocator = SegmentAllocator.implicitAllocator();
```

Comment:
Note to self: I should probably make a class with this, and the linker, which are common to all OS's, as static variables.